components/fxa-client/ios/FxAClient/FxAccountDeviceConstellation.swift

```diff
@@ -87,18 +92,29 @@ public class DeviceConstellation {
     }
 
     /// Send an event to another device such as Send Tab.
-    public func sendEventToDevice(targetDeviceId: String, e: DeviceEventOutgoing) {
+    public func sendEventToDevice(targetDeviceId: String,
+                                  _ e: DeviceEventOutgoing,
```

Comment:
```suggestion
                                  e: DeviceEventOutgoing,
```

I'm so sorry, @lougeniaC64; I realized this would be a breaking change 🤦🏼‍♀️ I thought it would be OK because we'd have to take a breaking change for a non-optional completion handler, anyway; but now that the completion handler is optional, removing the label would be the only breaking change.

Reply:
Yeah, I was wondering about that but I kinda deferred to you 😬 🫠 . I think I'm going to revert this label change just to be safe.